chispa/schema_comparer.py

```diff
@@ -44,33 +50,33 @@
 # deprecate this
 # perhaps it is a little faster, but do we really need this?
 # I think schema equality operations are really fast to begin with
-def assert_basic_schema_equality(s1, s2):
+def assert_basic_schema_equality(s1: StructType, s2: StructType) -> None:
     if s1 != s2:
         t = PrettyTable(["schema1", "schema2"])
         zipped = list(zip_longest(s1, s2))
         for sf1, sf2 in zipped:
             if sf1 == sf2:
-                t.add_row([blue(sf1), blue(sf2)])
+                t.add_row([blue(str(sf1)), blue(str(sf2))])
             else:
                 t.add_row([sf1, sf2])
         raise SchemasNotEqualError("\n" + t.get_string())
 
 
 # deprecate this.  ignore_nullable should be a flag.
-def assert_schema_equality_ignore_nullable(s1, s2):
+def assert_schema_equality_ignore_nullable(s1: StructType, s2: StructType) -> None:
     if not are_schemas_equal_ignore_nullable(s1, s2):
         t = PrettyTable(["schema1", "schema2"])
         zipped = list(zip_longest(s1, s2))
         for sf1, sf2 in zipped:
             if are_structfields_equal(sf1, sf2, True):
-                t.add_row([blue(sf1), blue(sf2)])
+                t.add_row([blue(str(sf1)), blue(str(sf2))])
             else:
                 t.add_row([sf1, sf2])
         raise SchemasNotEqualError("\n" + t.get_string())
 
 
 # deprecate this.  ignore_nullable should be a flag.
```

Comment:
Let's, maybe, add a deprecation warning?

Reply:
There's multiple functions with `# deprecate this` comment. Let's leave solving that for a follow-up PR.